manifests/d/Discord/Discord/Development/x86/1.0.771/Discord.Discord.Development.x86.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Discord.Discord.Development.x86
+PackageVersion: 1.0.771
+InstallerType: exe
+Scope: user
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install

```

```suggestion
UpgradeBehavior: deny
RequireExplicitUpgrade: true
```
These must be set due to how Discord's auto-updater works